aroquev/src/main/webapp/script.js

```diff
@@ -346,3 +346,86 @@ function deleteAllComments() {
     getServletComments();
   });
 }
+
+function onDOMLoad() {
+  buildWriteCommentsBox();
+  getServletComments();
+}
+
+async function checkLoggedIn() {
+  const response = await fetch('/login-status');
+  const loginInfo = await response.json();
+  return loginInfo;
+}
+
+async function buildWriteCommentsBox() {
+  const writeCommentsBox = document.getElementById('js-write-comment-box');
+  const loginInfo = await checkLoggedIn();
+  if (loginInfo.isLoggedIn) {  
+    const commentForm = document.createElement('form');
+    commentForm.action = 'javascript:;';
+    commentForm.addEventListener('submit', () => {
+      submitComment(commentForm);
+    });
+
+    // Build elements inside form
```

Comment:
This might be a slick solution to creating a lot of static HTML content!

https://stackoverflow.com/questions/494143/creating-a-new-dom-element-from-an-html-string-using-built-in-dom-methods-or-pro/35385518#35385518

Reply:
Cool! Done.